### ISSUE TITLE
Bug - When screen is small, a white overlay is added

### ISSUE THREAD

**Issue by olivermc94** [scania/corporate-ui#602]
**Describe the bug**  
When screen is small, a white overlay is added

**Screenshots**  
![image](https://user-images.githubusercontent.com/18164086/99090232-a6641680-25ce-11eb-88cb-3ca5e94b6fd0.png)

**Version of Corporate-ui**  
 4.2.0-2

**Comment by mmexvr** [comment]
We need example how to replicate this error. Please don't skip the question added in the bug report, it gives us more information how to debug it.

For example here it works. 
https://codepen.io/corporate-ui/pen/OYmqpr

Thanks for feedback!

**Comment by olivermc94** [comment]
This issue is visible when the screen is small and apparently when we dont have this 'Global' link in our header as the example you gave me on codepen.io. When i remove it through the inspector, i can see the same white space. 
![image](https://user-images.githubusercontent.com/18164086/99246245-44d9bd00-2805-11eb-8985-a379e4226117.png)

When removing the element link:
Without opacity
![image](https://user-images.githubusercontent.com/18164086/99246790-2a541380-2806-11eb-814f-ecf54ffd472a.png)

With opacity
![image](https://user-images.githubusercontent.com/18164086/99246856-435cc480-2806-11eb-918a-763db7c47b63.png)

**Comment by mmexvr** [comment]
Thanks for the replication, yeah, this is probably related with the global link taking up the rest of the space, to "cover" it. 

Thanks again!

**Comment by helloimela** [comment]
@olivermc94 That is correct, and expected style for navigation right now. When you click on the navigation hamburger menu, navigation will open, the navigation itself will cover up the whole page on mobile and content should be hidden. When you click the hamburger menu again, navigation is closed, and user can see the content. I think it is a similar behavior in Corporate UI 3 as well. But as we mention before, the interaction and UX for navigation is receiving some updates. But for now, that is the correct behavior. Screenshot below shows how it should looks like on mobile:

Navigation close:
![image](https://user-images.githubusercontent.com/1199101/99249299-2de99980-280a-11eb-80f8-f2dae11b4269.png)

Navigation open:
![image](https://user-images.githubusercontent.com/1199101/99249370-4eb1ef00-280a-11eb-97e0-80e7c7cdc49e.png)
